### PR TITLE
Updated Android example to latest Gradle plugin and API level

### PR DESCRIPTION
**Pull request by Nimrodda**
The example was using very old Android Gradle plugin and Gradle versions.
I updated those and also increased the compile and target SDK to point to API 29 which is the latest stable.
I also renamed the activity class to be consistent with Java class naming (HelloworldActivity -> HelloWorldActivity).
In addition, I made a small improvement to the Gradle config in gradle.properties.

### CODE REVIEW

File: examples/android/helloworld/app/.gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 /build
+.cxx/
```
Comment (dapengzhang0) [inline examples/android/helloworld/app/.gitignore:2]: Why adding this? Anything producing this directory?

**Review by dapengzhang0 — APPROVED** [review]
LGTM
cc the author of this example @ericgribkoff

**Comment by dapengzhang0** [comment]
@Nimrodda Please resolve merge conflict.

File: examples/android/helloworld/app/.gitignore (hunk 1 above)
Comment (Nimrodda) [reply examples/android/helloworld/app/.gitignore:2]: Yes, this is cmake output directory. See screenshot below:
image

**Comment by Nimrodda** [comment]
> @Nimrodda Please resolve merge conflict.

Merge conflicts are now resolved 👍

**Comment by Nimrodda** [comment]
@ericgribkoff @dapengzhang0 any updates on this PR?

**Pull request closed without merging**